src/node/in_memory.rs

```diff
@@ -1425,6 +1454,17 @@ impl<S: ForkSource + std::fmt::Debug + Clone> InMemoryNode<S> {
     /// Runs L2 transaction and commits it to a new block.
     pub fn run_l2_tx(&self, l2_tx: L2Tx, execution_mode: TxExecutionMode) -> Result<(), String> {
         let tx_hash = l2_tx.hash();
+
+        tracing::info!("");
+        tracing::info!("Validating {}", format!("{:?}", tx_hash).bold());
```

Comment:
question: Did we mean to keep this as an `info` level log? This feels more like `debug` as we already log out the transaction hash